src/seeder/task-seeder.ts

```diff
@@ -0,0 +1,55 @@
+import { CreateTaskRequest } from './../types/dto/tasks.dto'
+import * as faker from '@faker-js/faker'
+
+import { WorkflowStateResponse } from './../types/dto/workflowStates.dto'
+
+const apiUrl = 'http://localhost:3000'
+
+const token = ''
+
+async function getAllWorkflowStates(): Promise<WorkflowStateResponse[]> {
+  const res = await fetch(`${apiUrl}/api/workflow-states?token=${token}`)
+
+  const data = await res.json()
+
+  return data.workflowStates
+}
+
+async function createTask(payload: CreateTaskRequest) {
+  try {
+    await fetch(`${apiUrl}/api/tasks?token=${token}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(payload),
+    })
+  } catch (e) {
+    console.log('Error:', e)
+  }
+}
+async function seedTasks(count: number, workflowStateId: string) {
+  for (let i = 0; i < count; i++) {
+    const title = faker.fakerEN.lorem.sentence()
+    console.log(`Seeding ${i + 1} of ${count} for state ID ${workflowStateId}`)
+    await createTask({ title, workflowStateId: workflowStateId })
```

Comment:
This will seed data one by one for hundreds of tasks - since we have a very generous connection limit for dbs, we should attempt to run multiple insertions in parallel here.